test/sanity/code-smell/no-unicode-literals.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+UNICODE_LITERALS_USERS=$(grep -r unicode_literals . \
+    --exclude-dir .git \
+    --exclude-dir .tox \
+    --exclude no-unicode-literals.sh \
+    --exclude ansible-test-sanity-no-unicode-literals.json \
```

Comment:
Instead of excluding this by filename, I recommend excluding the entire `test/results/` directory.

Also, you'll need to exclude the docs for this feature as it's complaining about its own docs.

Reply:
I wasn't able to figure out how to exclude just ```test/results``` easily.  It seems that grep's ```--exclude-dir```  is for a specific directory name, not for a full path.  So I could ```--exclude-dir results``` but not ```--exclude-dir test/results```.  I could pipe it to a second grep I suppose.